public/js/CoinWrapperBuySignal.js

```diff
@@ -41,7 +41,8 @@ class CoinWrapperBuySignal extends React.Component {
     } = symbolConfiguration;
 
     const buyGridRows = gridTrade.map((grid, i) => {
-      return (
+      const modifiedGridTradeIndex = Math.min( Math.max ( currentGridTradeIndex , 7 ) , gridTrade.length - 4 );
```

Comment:
I am not sure about this change.

Are you saying you have a grid settings of more than 10? Wow...
I am not sure there are traders who are using that many grid settings.

Ok, let's say you do have lots of grid settings, the below screenshot made me so confused "Where is my Grid 11?"

<img width="329" alt="Screen Shot 2022-08-02 at 22 08 44" src="https://user-images.githubusercontent.com/5715919/182371119-920bd574-cb94-4b91-b2c3-1b932c0a4289.png">

Is it possible you can display a collapsed indicator such as `...5 grid trade hidden...` for hidden grid settings?
And when click the column, it shows the hidden columns. 

I just don't want to hide without any indicator.

I know I am asking for more changes (So sorry 🙏). If you don't have time, let's keep it original.
Again, sorry to ask for changes. @uhliksk

Reply:
I'm actually using 200 grid trades. I wanted the change in the code as simple as possible but I can add the indicator.

Reply:
Screenshot I put in PR section is the real thing. I just wanted to save resources not rendering the whole thing even when collapsed.

![image](https://user-images.githubusercontent.com/37454226/182350486-9b098ecc-5c2e-4bed-b1d9-bd9dd771c56a.png)

Reply:
Is this OK?

![image](https://user-images.githubusercontent.com/37454226/182441804-1bf1bb84-18ec-492e-8f60-898984d4efcf.png)

Reply:
@chrisleekr Did you reviewed the last screenshot?

Reply:
Grids with two indicators are taller than grids with single indicator. I'll fix it today to keep grid height consistent.

![image](https://user-images.githubusercontent.com/37454226/182662532-8165147e-6682-4a36-a5cc-74b603bd6251.png)

Reply:
Fixed. There are always 10 rows visible including indicators to keep the grid trade height uniform.

![image](https://user-images.githubusercontent.com/37454226/182674228-20baa7b2-5f01-4089-9e2e-3759beebb22e.png)

Reply:
@uhliksk 

Looks good. :+1: Thanks for the update.

Good to merge?

Reply:
@chrisleekr Yes, I think everything is good to go now. Thank you.